README.md

```diff
@@ -498,13 +498,13 @@ The [Google Places Search API](https://developers.google.com/places/web-service/
 #### LocationIQ (`:location_iq`)
 
 * **API key**: required
-* **Quota**: 6 request/second (30k req/day), then [ability to purchase more](http://locationiq.org/#pricing)
+* **Quota**: 60 requests/minute (2 req/day, 10k req/day), then [ability to purchase more](http://locationiq.org/#pricing)
 * **Region**: world
 * **SSL support**: yes
-* **Languages**: ?
-* **Documentation**: http://locationiq.org/#docs
+* **Languages**: worldwide
```

Comment:
Does this mean every language on earth is supported? Is it possible to be more specific? This can be critical info when it comes to deciding on an API.

Reply:
LocationIQ is a Nominatim (OpenStreetMap) service. You can safely assume it's a mirror. OpenStreetMap certainly doesn't support every language on Earth unless contributors add them!

I'll revert `worldwide` back to `?`.